src/prefect/cli/flow_run.py

```diff
@@ -164,6 +165,12 @@ async def logs(
         ),
         min=1,
     ),
+    reverse: bool = typer.Option(
+        None,
```

Comment:
I think this default should be `False`?

Let me know if you need help with the tests!

Reply:
well noted I agree default value must be false since reverse is a boolean type